tests/postgres_creds.json

```diff
@@ -0,0 +1,6 @@
+{
```

Comment:
does cloud automation need to change at all to support splitting this stuff out?

Reply:
no, the wts config handles both `SQLALCHEMY_DATABASE_URI` and `POSTGRES_CREDS_FILE` and cloud-automation already uses the later

i split it out to make it easier to migrate the test DB

Reply:
actually i didn't need to do that, i reverted this part, thanks